smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/httpResponse/bindingTraits/HTTPResponseTraitQueryParams.kt

```diff
@@ -1,30 +0,0 @@
-/*
- * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
- * SPDX-License-Identifier: Apache-2.0.
- */
-
-package software.amazon.smithy.swift.codegen.integration.httpResponse.bindingTraits
```

Comment:
This used to render a statement to set a member to nil if it is in the response and has the `@httpQuery` trait.  This was incorrect.

Instead, this renderer is removed and a member with the `@httpQuery` trait will be rendered to the response like any other member.